src/com/google/enterprise/adaptor/fs/FsAdaptor.java

```diff
@@ -668,14 +683,13 @@ private Path getDfsRoot(Path path) throws IOException {
 
   /** Verify the path is available and we have access to it. */
   private void validateShare(Path sharePath) throws IOException {
-    if (delegate.isDfsNamespace(sharePath)) {
```

Comment:
Can we add note to javadoc what are the valid things that this method can be called on.